.github/workflows/container-sync.yml

```
@@ -21,6 +21,10 @@ on:
         description: Whether to sync images for Zed and newer
         type: boolean
         default: true
+      skip_pulp:
+        description: Whether to skip pulp promotion
```

Applies to other inputs
```suggestion
      skip_test_pulp:
        description: Whether to skip test pulp promotion
```

---

Changed.